_posts/2020-06-07-turmite-arena.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Turmite Arena"
+date: 2020-06-07 23:00
+categories: math automata
+---
+
+{::options parse_block_html="true" /}
+
+# Turmite Arena app
+{: #app-section}
+
+{% include collapser.markdown %}
+
+<div id="game-window">
+
+<div class="no-js">
+This app requires JavaScript to run.
+</div>
+
+## Canvas
+{: #turmite-canvas-section}
+
+{% include collapser.markdown %}
+
+<div id="turmite-canvas"></div>
+
+## Messages
+{: #turmite-message-section}
+
+<div id="turmite-message"></div>
+
+## Status
+{: #turmite-status-section}
+
+<div id="turmite-status">
+
+Not initialized
+
+</div>
+
+## Settings
+{: #turmite-settings-section}
+
+{% include collapser.markdown %}
+
+<div id="turmite-settings-form">
+<input type="button" value="Pause/resume" onclick="turmites.sendPauseResume()">
+
+<input type="button" value="Apply settings and restart session" onclick="turmites.sendRestart()">
+
+<label for="settings-seed">Random seed</label>
+<input type="text" id="settings-seed" name="settings-seed">
+<input type="button" value="Generate a new seed" onclick="turmites.generateNewSeed()">
+
+<label for="settings-canvas-w">Canvas width</label>
+<input type="number" id="settings-canvas-w" name="settings-canvas-w" min="1" max="10000000">
+
+<label for="settings-canvas-h">Canvas height</label>
+<input type="number" id="settings-canvas-h" name="settings-canvas-h" min="1" max="10000000">
+
+<label for="settings-ncolors">Number of colours</label>
+<input type="number" id="settings-ncolors" name="settings-ncolors" min="1" max="64">
+
+<label for="settings-nstates">Number of states</label>
+<input type="number" id="settings-nstates" name="settings-nstates" min="1" max="10000000">
+
+<label for="settings-nturmites">Number of turmites</label>
+<input type="number" id="settings-nturmites" name="settings-nturmites" min="1" max="10000000">
+
+<label for="settings-ffiters">Number of steps per tick</label>
+<input type="number" id="settings-ffiters" name="settings-ffiters" min="1" max="10000000">
+
+<label for="settings-tick-delay">Tick delay (ms)</label>
+<input type="number" id="settings-tick-delay" name="settings-tick-delay" min="10" max="10000">
+
+Colour palette
+
+<input type="radio" id="settings-color-map-handpicked" name="settings-color-map" value="handpicked">
+<label for="settings-color-map-handpicked">Handpicked</label>
+
+<input type="radio" id="settings-color-map-sobol" name="settings-color-map" value="sobol">
+<label for="settings-color-map-sobol">Sobol</label>
+
+<input type="radio" id="settings-color-map-random" name="settings-color-map" value="default">
+<label for="settings-color-map-random">Random</label>
+</div>
+
+</div>
+
+# About Turmite Arena
+{: #about-game}
+
+{% include collapser.markdown %}
+
+<div>
+
+## Inspiration
```

Comment:
Speaking about your inspiration in my opinion is a good hook into explaining the actual project 🐜